misc/pytorch_toolkit/chest_xray_screening_federated_gcn/configs/fl_without_gnn.json

```diff
@@ -0,0 +1,38 @@
+{
+    "train": {
+        "data": "/storage/adityak/chest_x_ray/",
+        "split_npz": "/storage/adityak/split.npz",
```

Comment:
Change to a relative path. Make the changes to all the applicable places in this file.